README.md

```diff
@@ -561,6 +561,130 @@ Use it to type in your key and install.
+        <strong>Notes:</strong>
+        <ul>
+          <li>First fork to get long!</li>
+          <li>Very quick stop-and-go response</li>
+          <li><code>alexandresato/extract_secoc_key_btn</code> includes a TSK key extract button and is rebased with <code>personal3</code> often.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>sunnypilot/master-new</code> <br/> 
+        (a.k.a SunnyPilot) <br/>
+        <a href="https://discord.com/invite/sunnypilot">SunnyPilot discord</a>
+      </td>
+      <td>
+        <strong>Lat:</strong> Yes from upstream <br/>
+        <strong>MADS:</strong> Yes from community (MADS original author) <br/>
+        <strong>Long:</strong> No <br/> <br/>
+        <strong>Notes:</strong>
+        <ul>
+          <li>Model switcher to easily switch between various models</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>chrispypatt/sunnypilot</code>  <br/> 
+        (fork of SunnyPilot)
+        <a href="https://discord.com/invite/sunnypilot">SunnyPilot discord</a>
+      </td>
+      <td>
+        <strong>Lat:</strong> Yes from upstream <br/>
+        <strong>MADS:</strong> Yes from community (MADS from SP) <br/>
+        <strong>Long:</strong> Yes from community (original author) <br/> <br/>
+        <strong>Notes - same as <code>sunnypilot/master-new</code> plus:</strong>
+        <ul>
+          <li>Includes a TSK keyboard with key caching</li>
+          <li><code>commaai/master</code> and most forks based on it have unstable experimental mode due to the latest model. The model switcher in SP allows you to use old models with stable experimental mode</li>
+          <li>Maintained as best effort. This may not always be up to date with latest SunnyPilot. Reach out to @Chrispypatt for requests to rebase his fork with SunnyPilot's rewrite.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>optskug/SiennaFP</code> <br/> 
+        (fork of FrogPilot) <br/>
+        <a href="https://discord.com/invite/frogpilot">FrogPilot discord</a>
+      </td>
+      <td>
+        <strong>Lat:</strong> Yes from community (from anrum) <br/>
+        <strong>MADS:</strong> Yes from community (AOL from FP) <br/>
+        <strong>Long:</strong> No <br/> <br/>
+        <strong>Notes</strong>
+        <ul>
+          <li>anrum's old fork of FP and first fork to support TSK lateral!</li>
+          <li>Includes a TSK keyboard with key caching</li>
+          <li>Includes auto key installer</li>
+          <li>Uses an old AGNOS version. When downgrading, OP may get stuck in a registration loop. In this case, tap-tap-tap on the boot logo and reset the device to recover and then install again.</li>
+        </ul>
+      </td>
+    </tr>
+</table>
+
+> [!WARNING]
+> `optskug/SiennaFP` is the only FrogPilot fork/branch with TSK support. Do not install the latest FP unless FrogGoesMoo confirmed that it supports TSK vehicles.
```

Comment:
Frog just pushed a change in staging that claims to support TSK. Need to check.

Reply:
If you want to merge, let's remove this warning

Reply:
We can just take this out if you want to merge

Reply:
https://discord.com/channels/1137853399715549214/1137905508217540699/1331156376336076840

Reply:
I think until we confirm it's working we should hold off on removing this